### PR TITLE
Webhook for LRS (trigger on verb id) 

### PR DESCRIPTION
**Pull request by adurolms**
Screenshots on this function:

Manage page

Manage page

### CODE REVIEW

File: app/locker/repository/Statement/EloquentStorer.php [hunk 1]
```diff
@@ -41,7 +43,14 @@ public function store(array $statements, array $attachments, StoreOptions $opts)
     $this->voider->voidStatements($statements, $opts);
     $this->attacher->store($attachments, $this->hashes, $opts);
     $this->activateStatements($ids, $opts);
-
+    // Check if there's any webhook set up to queue up WebhookAsyncReq job
+    $lrsId = (string) $opts->getOpt('lrs_id');
+    $webhook = Webhook::where('lrs_id', $lrsId)->first();
```
Comment (ryansmith94) [inline app/locker/repository/Statement/EloquentStorer.php]: @ht2 please confirm that this is ok (converts a MongoId to a String for filtering).

**Comment by ryansmith94** [comment]
Hi @adurolms, thanks for the pull request, we'll check this out shortly.

**Comment by ryansmith94** [comment]
Unfortunately we're all busy this week, so unless someone gets a spare minute this won't be seen until Monday. In the meantime, it'd be really good if you could add some tests to this pull request. Providing screenshots of the UI may also speed up the review process.

**Comment by adurolms** [comment]
hi @ryansmith94 , thanks for inform, i just provided the screenshot, will do unit test later

**Comment by ryansmith94** [comment]
No worries @adurolms. Thanks for the screenshots. Give me a shout when you've done the unit testing :+1:

**Comment by ryansmith94** [comment]
Any update on this @adurolms?

**Comment by jamarr81** [comment]
Would really like to see support for webhooks added to LL. We are very interested in using LL for an upcoming project. We would much prefer to use a LL side event-based trigger as opposed to a client side polling technique which is much less efficient. How do people currently setup their statement analytics? direct connection to the underlying MongoDB service? a message bus that can broadcast statements to multiple services?

**Comment by ryansmith94** [comment]
What do you want to do about this @andrewhickey?

**Comment by nadavkav** [comment]
+1

**Comment by ryansmith94** [comment]
V2 is very likely to have webhooks. Stay tuned to the [LL chat](gitter.im/LearningLocker/learninglocker) for its release in the next few months.

**Comment by nadavkav** [comment]
OT: @ryansmith94, Is there any alpha/beta for V2 we can test before the scheduled release?

**Comment by ryansmith94** [comment]
Not currently, but if we do release one I'll post that information on the LL chat.

**Comment by nadavkav** [comment]
Thanks

**Pull request closed without merging**